src/V3OrderParallel.cpp

```diff
@@ -1737,6 +1737,34 @@ class DpiImportCallVisitor final : public VNVisitor {
     VL_UNCOPYABLE(DpiImportCallVisitor);
 };
 
+//######################################################################
+// DpiThreadsVisitor
+
+// Get number of threads occupied by this mtask
+class DpiThreadsVisitor final : public VNVisitorConst {
+    int m_threads = 1;  // Max number of threads used by this mtask
```

Comment:
```suggestion
    uint32_t m_threads = 1;  // Max number of threads used by this mtask
```

Or change the method below that returns a uint32_t.